starter/script.js

```diff
@@ -1 +1,54 @@
-// put code here!
+// put code here
+terrain();
+
+function terrain() {
+var terrain = prompt("Choose your terrain: (road, trail)");
+  if (terrain == "road") {
+    RoadStory();
+  }
+  else if (terrain === "trail") {
+    TrailStory();
+    }
+  }
+
+function RoadStory(){
```

Comment:
function names with a capital letter usually indicate constructor functions - we'll learn more about this next next week!